### DISCUSSION TITLE
Is it possible to anable font ligatures in cmder?

### DISCUSSION THREAD

**Discussion by VictorSilvano** [cmderdev/cmder#2623] [Q&A] [answered]
hey I'm really new to tech in general and still getting my bearings. I've found in other programs I can enable ligatures which, at least for me, really cleans up the code I'm writing, is it possible to enable it in cmder?

P.S thanks for making this amazing tool its made learning so much easier :D

**Comment by t5k6** [comment, marked as answer]
cmder uses ConEmu, and it seems it doesn't support ligatures yet.
https://github.com/Maximus5/ConEmu/issues/703

**Reply by VictorSilvano** [reply]
Thanks so much, that really helps me understand appreciate it :D

**Comment by DRSDavidSoft** [comment]
As an alternative, in the meantime you can use [Windows Terminal](https://github.com/cmderdev/cmder/wiki/Seamless-Windows-Terminal-Integration) instead of ConEmu as the terminal window for Cmder.  It has support for font ligatures, colorful emojis, and a blurred-behind background.
